desktop/src/client/proxy/manual-proxy-configuration-window/manual-proxy-configuration-window.html

```diff
@@ -5,21 +5,22 @@
     <link href="./manual-proxy-configuration-window.css" rel="stylesheet" type="text/css">
 </head>
 <script>
-    const { ipcRenderer, remote } = require("electron");
-    const initial = remote.getCurrentWindow().currentSettings;
+    const { ipcRenderer } = require("electron");
 
-    document.addEventListener('DOMContentLoaded', function () {
-        window.urlEl = document.getElementById("url");
-        window.portEl = document.getElementById("port");
-        window.usernameEl = document.getElementById("username");
-        window.passwordEl = document.getElementById("password");
+    ipcRenderer.invoke("proxy-configuration-requested").then((settings) => {
+        document.addEventListener("DOMContentLoaded", () => {
```

Comment:
We should check `document.readyState` to make sure loading hasn't completed and we've missed the DOMContentLoaded event.

Reply:
See: https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event#checking_whether_loading_is_already_complete